docs/blog/2023-02-17-how-to-use-drand.md

```diff
@@ -119,16 +119,18 @@ const chain = new HttpCachingChain("https://api.drand.sh")
 const client = new HttpChainClient(chain)
 fetchBeaconByTime(client, Date.now()).then(beacon => {   
 
-	const flip = parseInt(Number("0x" + beacon.randomness)) % 2    
+	const flip = BigInt("0x" + beacon.randomness) % BigInt(2)
 
-	if (flip === 0) {     
+	if (flip === 0) {      
 		console.log(beacon.round+": HEADS!")
 	} else {     
 		console.log(beacon.round+": TAILS!")   
 	} 
 })
 ```
 
+A small note: we have to use `BigInt` here, because javascript only supports 53 bit integers safely! Additionally, because drand mainnet emits randomness every 30 seconds, you will see the same output for multiple runs until the newest round is emitted.
```

Comment:
```suggestion
A small note: we have to use `BigInt` here, because javascript numbers only support 53 bit integers safely! Additionally, because drand mainnet emits randomness every 30 seconds, you will see the same output for multiple runs until the newest round is emitted.
```